lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -24,7 +24,7 @@ defmodule Apr.Views.CommerceOrderSlackView do
       {"high_risk", "approved", %{"items_total_cents" => cents}}
         when cents >= 10_000_00 ->
           generate_slack_message(event, routing_key)
-      {"high_risk_async_payment", "processing_approval", %{"items_total_cents" => cents}}
+      {"high_risk", "processing_approval", %{"items_total_cents" => cents}}
```

Comment:
thought: I'm wondering if there is any way to combine this line with the above.

Reply:
`{"high_risk", ["approved", ["processing_approval"], %{"items_total_cents" => cents}}` doesn't work.

Reply:
Not exactly sure. 🤔  Maybe something like:
```elixir
{"high_risk", event, %{"items_total_cents" => cents}}
  when event in ["approved", "processing_approval"] and cents >= 10_000_00 ->
    generate_slack_message(event, routing_key)
```

Reply:
Thank you for the suggestion! Changed it here: https://github.com/artsy/aprd/pull/380/commits/7b147aa5dddacf9d9ab7ec87927c4c59376fe1b1